.github/workflows/release.yaml

```diff
@@ -41,8 +76,8 @@ jobs:
           name: Release ${{ steps.get_version.outputs.VERSION }}
           files: otel-java-lambda-wrapper.jar
```

Comment:
Consider adding `draft: true` to have a chance to edit release notes before publishing.

Reply:
Good idea!